EpsilonLights/Src/main.c

```diff
@@ -287,12 +287,29 @@ static void MX_CAN2_UserInit(void)
     sFilterConfig.FilterActivation = ENABLE;
     sFilterConfig.BankNumber = 0; // Set all filter banks for CAN2
 
+    CAN_FilterConfTypeDef batteryFilterConfig;
```

Comment:
Why is a second filter necessary?

Reply:
Here's the explanation:
https://github.com/UCSolarCarTeam/Epsilon-Embedded-Software/issues/40